static/scss/certificates/certificate.scss

```diff
@@ -289,3 +275,144 @@ $text-gray-light: #d2d0d1;
     width: 25px;
   }
 }
+
+@media print {
+  a {
+    color: black !important;
+  }
+
+  .notifications.site-wide {
+    display: none !important;
+  }
+
+  .no-print {
+    display: none;
+  }
+
+  .full-width-print {
+    max-width: 100%;
+  }
+
+  .certify-by-row {
+    display: block;
+
+    &:after {
+      content: "";
+      display: block;
+      clear: both;
+    }
+
+    .certify-by {
+      float: left;
+      width: 33.3333%;
+
+      &.signature-1 {
+        float: none;
+        margin: 0 auto;
+      }
+
+      &.signature-2 {
+        width: 30%;
+
+        &:first-child {
+          margin-left: 20%;
+        }
+      }
+
+      &.signature-4 {
+        width: 25%;
+      }
+
+      &.signature-5 {
+        width: 20%;
+      }
+    }
+  }
+
+  @page {
+    size: auto;
+  }
+
+  .certificate-page {
+
+    .certificate-logo {
+      max-width: 170px;
+      margin: 0 auto 10px;
+
+    }
+
+    .certificate-wrapper {
+      padding: 0;
+
+      .certificate {
+        border: 0;
+        padding-top: 20px;
+
+        .institute-logo {
+          max-width: 140px;
+          margin-bottom: 10px;
+        }
+      }
+
+      .certificate-holder {
```

Comment:
In the print style sheet, I don't see any reason to limit the width of the `certificate-holder` div to 730px.

Reply:
This is a nice to have change.

Reply:
un-setting the max-width in print will cause that UI..

<img width="1279" alt="Screen Shot 2019-09-02 at 2 55 28 PM" src="https://user-images.githubusercontent.com/7334669/67573664-4ea00680-f752-11e9-9e93-8d3f3baebf87.png">

I believe, you are thinking something like inside print css: 

```
.certificate-holder {
        max-width: unset;
        .certify-by {
 

```